AUTHORS

```diff
@@ -378,7 +377,7 @@ Eric Miller <emiller42@gmail.com>
 Venkata Ramana <idlike2dream@gmail.com>
 Cody Herbst <cyherbst@gmail.com>
 Nishith Shah <nishithshah.2211@gmail.com>
-AMiT Kumar <dtu.amit@gmail.com>
+Amit Kumar <dtu.amit@gmail.com>
```

Comment:
We allow abbreviations so perhaps we should allow non-traditional casing if that's how the person represents their name. Did you check with Amit to see if this is his preferred "signature"?